ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/SupportsPeras.hs

```diff
@@ -66,3 +70,14 @@ instance StandardHash blk => BlockSupportsPeras blk where
 
   perasCertRound = pcCertRound
   perasCertBoostedBlock = pcCertBoostedBlock
+
+instance Serialise (HeaderHash blk) => Serialise (PerasCert blk) where
+  encode PerasCert{..} =
```

Comment:
Nitpick: The [style guide](https://ouroboros-consensus.cardano.intersectmbo.org/docs/howtos/contributing/style_guide) asks us to not use `RecordWildCards`. (I personally like them though :sweat_smile: And there are still lots of usages left from before this rule existed.)